sound/soc/soc-pcm.c

```diff
@@ -99,6 +100,7 @@ void snd_soc_runtime_deactivate(struct snd_soc_pcm_runtime *rtd, int stream)
 		codec_dai->active--;
 	}
 }
+EXPORT_SYMBOL_GPL(snd_soc_runtime_deactivate);
```

Comment:
these two exports need to be separate patch.

Reply:
done